src/Context/DrupalUtilsApiContext.php

```diff
@@ -49,18 +49,19 @@ public function visitNodeByTitle($type, $title) {
   }
 
   /**
-   * Visit node edit page by title.
+   * Visit node custom page by title.
+   * Custom page includes edit, layout and revisions.
    *
-   * @When I visit :type edit page with title :title
+   * @When I visit :type titled :title :page page
    */
-  public function visitNodeEditByTitle($type, $title) {
+  public function visitNodeCustomPageByTitle($type, $title, $page) {
```

Comment:
We need to watch out and make sure this is not a backwards incompatible change. We can easily do that by setting a sane default for $page and make it optional. See some step definitinos in MinkElementTrait for an example on how to do do that.
Thus:
"When I visit :type :title" must remain working

For the other variant I'd suggest improving the language by doing:
@When I visit :page page of :type :title
--> When I visit layout page of article foo.
(I think the order does not matter).